tools/gluten-te/centos/defaults.conf

```diff
@@ -43,10 +51,14 @@ DEFAULT_DOCKER_TARGET_IMAGE_BUILDENV=gluten-te/gluten-buildenv
 DEFAULT_DOCKER_TARGET_IMAGE_BUILD=gluten-te/gluten-build
 
 # Docker registry used to pull pre-built images to speed-up builds
-DEFAULT_DOCKER_CACHE_REGISTRY=
+# DEFAULT_DOCKER_CACHE_REGISTRY=
+# for 10.0.0.25 test 
+DEFAULT_DOCKER_CACHE_REGISTRY=10.0.0.25:5000
 
 # Docker registry to push pre-built images
-DEFAULT_DOCKER_PUSH_REGISTRY=
+# DEFAULT_DOCKER_PUSH_REGISTRY=
+# for 10.0.0.25 test
+DEFAULT_DOCKER_PUSH_REGISTRY=10.0.0.25:5000
```

Comment:
We'd revert the change before merging? do I understand it correctly?

Reply:
I want to keep the settings of DEFAULT_DOCKER_PUSH_REGISTRY and DEFAULT_DOCKER_CACHE_REGISTRY in this place. Are there any problems with this?

 If revert changes, where should these variables be set?

Reply:
I plan to revert the change in defaults.conf and set DOCKER_PUSH_REGISTRY and DOCKER_CACHE_REGISTRY in .github/workflow/velox_be.yml